### PR TITLE
EDA-639 Send email when a status change requires notification

### PR DESCRIPTION
**Pull request by ricardocaif-eb**
Added notification for approval and denial. Change requiered notification is pending util method is defined.

### CODE REVIEW

File: supplier_app/__init__.py [hunk 1]
```diff
@@ -32,6 +32,17 @@
         'Please click on on the following link to register' + '\n' +
         settings.COMPANY_INVITATION_URL
     },
+    'taxpayer_approval': {
+        'subject': 'Your taxpayer has been approved',
+        'body': 'You are ready to start using BriteSu' + '\n' +
+        'you can access the platform now and upload your invoices' +
+        settings.SUPPLIER_HOME_URL
+    },
```
Comment (lautaro-eb) [inline supplier_app/__init__.py]: There is no need to concatenate this way ...BriteSu' + '\n' ... 
Also, can you use "{}".format(variable)?

File: supplier_app/__init__.py [hunk 2]
```diff
@@ -32,6 +32,17 @@
...
+    'taxpayer_denial': {
+        'subject': 'Your taxpayer has been rejected',
+        'body': 'We are affraid that the taxpayer you were trying to submit is invalid' + '\n' +
+        'Plese contact the eventbrite eployee that hired you'
```
Comment (lautaro-eb) [inline supplier_app/__init__.py]: the same here
Also:
affraid -> afraid
Plese -> Please
eployee -> employee

File: supplier_app/views.py [hunk 3]
```diff
@@ -254,6 +257,10 @@ def approve_taxpayer(self, taxpayer_id, request=None):
     taxpayer = TaxPayer.objects.get(pk=taxpayer_id)
     taxpayer.approve_taxpayer()
     taxpayer.save()
+    subject = email_notifications['taxpayer_approval']['subject']
+    body = email_notifications['taxpayer_approval']['body']
+    emails = get_user_emails_by_tax_payer_id(taxpayer_id)
+    send_email_notification(subject, body, emails)
```
Comment (lautaro-eb) [inline supplier_app/views.py]: It looks like you are sending emails in multiple flows, think and discuss the async implementation as a Team!

File: supplier_app/__init__.py (hunk 1 above)
Comment (ricardocaif-eb) [reply supplier_app/__init__.py]: Yes!

File: supplier_app/__init__.py (hunk 2 above)
Comment (ricardocaif-eb) [reply supplier_app/__init__.py]: Thank you!

File: supplier_app/views.py (hunk 3 above)
Comment (ricardocaif-eb) [reply supplier_app/views.py]: Created a spike to investigate and discuss about it to implement it on the project.
[EDA-672](https://jira.evbhome.com/browse/EDA-672)

Comment (lautaro-eb) [reply supplier_app/views.py]: Nice

**Pull request merged**